docs/reference/expectations/result_format.md

```diff
@@ -96,112 +93,35 @@ cases for working with Great Expectations, including interactive exploratory wor
 |    SUMMARY                            | Detailed exploratory work with follow-on investigation.      |
 |    COMPLETE                           | Debugging pipelines or developing detailed regression tests. |
 
-## result_format examples
-
-Example input:
-```python
-print(list(my_df.my_var))
-['A', 'B', 'B', 'C', 'C', 'C', 'D', 'D', 'D', 'D', 'E', 'E', 'E', 'E', 'E', 'F', 'F', 'F', 'F', 'F', 'F', 'G', 'G', 'G', 'G', 'G', 'G', 'G', 'H', 'H', 'H', 'H', 'H', 'H', 'H', 'H']
-```
-
-Example outputs for different values of `result_format`:
+## Examples
 
+The following examples will use the data defined in the following Pandas DataFrame:
 
-```python
-my_df.expect_column_values_to_be_in_set(
-    "my_var",
-    ["B", "C", "D", "F", "G", "H"],
-    result_format={'result_format': 'BOOLEAN_ONLY'}
-)
-{
-    'success': False
-}
+```python name="pandas_df_for_result_format"
```

Comment:
Totally optional nitpick - do you think this dataframe can be reduced in size so that each of the columns can fit on a single line e.g. `"my_var": ["A", "B", "B", "C", "C", "D", ... <etc>],` instead of 
```
"my_var": [
            "A",
            "B",
            "B",
            "C",
            "C",
            "C",
            "D",
            "D",
            "D",
            "D",
            "E",
            "E",
            "E",
            "E",
            "E",
        ],
```